### PR TITLE
Converts events page to use modular scss

### PR DESCRIPTION
**Pull request by advaithg**
Title describes it all! 
A pretty short PR that's part of the incremental addressing of issue #191 

Also changes the commented out calendar for Fall and beyond + mentions the CSS changes made in a comment

### CODE REVIEW

File: styles/App.scss [hunk 1]
```diff
@@ -26,6 +26,6 @@
 /* pages */
 @import 'pages/index';
 @import 'pages/committees';
-@import 'pages/events';
+// @import 'pages/events';
```
Comment (mattxwang) [inline styles/App.scss:29]: Let's comment this out before merging!

Comment (advaithg) [reply styles/App.scss:29]: oh shoot I think that happened because of me pulling main back into the branch

Comment (advaithg) [reply styles/App.scss:29]: oh no it's still commented - I assume you meant remove, so I'll get rid of it

Comment (mattxwang) [reply styles/App.scss]: Yup sorry, mistyped!

**Review by mattxwang — APPROVED** [review]

**Pull request merged**